src/client/App.js

```diff
@@ -35,6 +36,7 @@ function App() {
           </AuthenticatedRoute>
         </Switch>
       </Router>
+      <MessageList />
```

Comment:
Dont apply the component! Just make it. Its someone elses job to activate it

Reply:
Done